examples/lib/defaultHtmlStepUi.js

```diff
@@ -118,7 +118,14 @@ function DefaultHtmlStepUi(_sequencer, options) {
               paramVal + '">' + '<span class="input-group-addon"><i></i></span>' +
               '</div>';
           }
-          else { // Non color-picker input types
+          else if(inputDesc.id === 'coordinates-input'){
```

Comment:
You can use `inputDesc.type` instead of `id` here.

Reply:
ok i will remove that.